### DISCUSSION TITLE
Adnade Issue

### DISCUSSION THREAD

**Discussion by andriansandi** [engageub/InternetIncome#101] [General]
I got the issue when using the newest branch main code, adnade container shows IP reached simultaneously sessions

![Screenshot 2024-02-01 215528](https://github.com/engageub/InternetIncome/assets/596269/678194f7-52c5-4feb-9c52-578c008ae9b4)

**Comment by engageub** [comment]
Hello,
Are you using shared IP address or dedicated IP address.
Are you using main branch or test branch.
A delay of 5 minutes is given already between each restart to avoid this.

Thank you

**Reply by andriansandi** [reply]
I'm using dedicated IP addresses and proxy

**Reply by engageub** [reply]
Did it happen in 1st run or you deleted and started the containers immediately.
If you deleted and started the containers it will happen as it may be immediate. It will automatically restart after one or two hours, then you can look into the containers. 
Thank you

**Reply by andriansandi** [reply]
I already deleted and started the containers persist. only adnade and ebesucher from this monday earning was dropped

**Reply by engageub** [reply]
Are you using test branch or main branch.
If you are using main branch, there have been no changes related to them in the past month.

Thank you

**Comment by andriansandi** [comment]
using the main branch

**Reply by engageub** [reply]
If you are using the same code, then no changes have been made to your code you are using. 
May be traffic has been reduced or users might have increased.

**Comment by andriansandi** [comment]
Hi @engageub , 

Is there any possibility that adnade PTP working with proxies?

**Reply by engageub** [reply]
Yes, it works with proxies but sometimes IP may also be blocked.
As surfbar earnings are high compared to PTP, it has been removed.
If you wish to use it, you may change the url in internetIncome.sh file

Thank you

**Reply by andriansandi** [reply]
how to make it working with proxies? Lastly, it have been worked well on multiips

**Reply by engageub** [reply]
You have to add proxies in proxies.txt . Please see https://github.com/engageub/InternetIncome?tab=readme-ov-file#how-to-use-residential-proxies


Regarding PTP link, you have to replace the existing url with PTP link in the code after downloading. https://github.com/engageub/InternetIncome/blob/main/internetIncome.sh#L328C110-L328C184

Existing url below
```
-e FF_OPEN_URL="https://adnade.net/view.php?user=$ADNADE_USERNAME&multi=4"
```
Replace it with the following

```
-e FF_OPEN_URL="https://adnade.net/ptp/?user=$ADNADE_USERNAME"
```


Regarding Multi IPs, are you asking me or telling me that it worked fine with Multi IPs.

Please use test branch if you want to use multi ips.

Thank you

**Reply by andriansandi** [reply]
Can I use ptp and surfbar simultaneously ?

**Reply by engageub** [reply]
Yes, you can but it requires two containers to be running. Sometimes, it does not run, showing message that a session is already running. So you have to watch out for this message. Opening in new tab, can be done but with the existing code it may not be possible. Modifications have to be made. 
You can create two different folders for InternetIncome script and change the url in the other.
There will be two proxy containers running for each IP in this case if you prefer to use same IPs in different folders.
If you use different IP address in different folders, there will not be any problem. 
Just make sure you don't run into loss as it consumes more CPU and memory compared to other containers. 

Thank you
